src/ocamlorg_web/lib/handler.ml

```diff
@@ -396,20 +387,29 @@ let package_docs t req =
   match version_opt with
   | None -> not_found req
   | Some version ->
-      let version = Ocamlorg_package.Version.to_string version in
-      let target = Ocamlorg_frontend.Url.package_doc package version in
+        let version_string = if Ocamlorg_package.is_latest_version t name version then 
+         "latest" else
+          Ocamlorg_package.Version.to_string version in
+      let target = Ocamlorg_frontend.Url.package_doc package version_string ~is_latest_url:false in
       Dream.redirect req target
 
 let package_versioned t kind req =
   let name = Ocamlorg_package.Name.of_string @@ Dream.param req "name" in
-  let version =
-    Ocamlorg_package.Version.of_string @@ Dream.param req "version"
+  let version_from_url = Dream.param req "version"
+  in
+  let package = if version_from_url = "latest" then 
+    Ocamlorg_package.get_package_latest t name
+  else 
+    let version = Ocamlorg_package.Version.of_string @@ version_from_url in
+     Ocamlorg_package.get_package t name version
   in
-  let package = Ocamlorg_package.get_package t name version in
   match package with
   | None -> not_found req
   | Some package ->
       let open Lwt.Syntax in
+      let _version = Ocamlorg_package.version package in
```

Comment:
underscore here means you can remove this line, since you never use the variable